src/mono/sample/wasm/browser/main.js

```diff
@@ -1,3 +1,5 @@
+var BINDING_GLOBAL;
+document.getElementById("demo").onclick = function() {click()};
```

Comment:
You can move this registration inside `onRuntimeInitialized` callback, in which you would get the `BINDING` already, so that you don't have to make it global.

Reply:
done.